doc/source/examples/rpt/particle-fem-reconstruction/particle-fem-reconstruction.rst

```diff
@@ -0,0 +1,170 @@
+
+    subsection detector parameters
+      set detector positions file         = positions.detector
+      set radius                          = 0.0381
+      set length                          = 0.0762
+      set dead time                       = 7.6215e-06,1.27187e-05,7.94972e-06,1.0988e-05
+      set activity                        = 6.01924e+06,7.09549e+06,7.24499e+06,7.75147e+06
+      set attenuation coefficient reactor = 0,0.50377,0,0.95902
+    end
+
+FEM reconstruction parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the subsection ``FEM reconstruction parameters``, we specify the file that contains the experimental counts and the export file for the reconstructed positions. Parameters used for the FEM reconstruction are described in the :doc:`../../../parameters/rpt/fem_reconstruction` documentation page.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # FEM reconstruction parameters
+    #---------------------------------------------------
+
+    subsection fem reconstruction
+      set mesh type                = dealii
+      set z subdivisions           = 2
+      set mesh refinement          = 2
+      set experimental counts file = experimental-counts.txt
+      set export positions file    = found_positions
+     set cost function type       = relative
+      set dof handler file         = temp_dof_handler.dof
+      set nodal counts file        = temp_nodal_counts_detector00.counts, temp_nodal_counts_detector01.counts, temp_nodal_counts_detector02.counts, temp_nodal_counts_detector03.counts
+    end
+
+:math:`C_{exp}`
+
+Running the simulation
+----------------------------------
+Assuming that ``rpt_l2_projection_3d`` and ``rpt_fem_reconstruction_3d`` executables are within your path, you can start launching FEM-PR by typing :
+
+.. code-block:: text
+
+    rpt_l2_projection_3d rpt-fem-reconstruction.prm
+
+This step using L2 projection technique smooth the noise of the Monte Carlo method and calculates the photon counts at the nodes.
+After this part is done, the program generates the file of nodal counts from each detector as such : ``temp_nodal_counts_detector00.counts``.
+Next step would be Launching the position reconstruction by:
+
+.. code-block:: text
+
+	rpt_fem_reconstruction_3d rpt-fem-reconstruction.prm
+
+
+Results
+--------
+With the figure shown above we assess the validity of the reconstruction algorithm with a reconstruction of a set of experimental points.
+It shows the real positions of the particle and the reconstructed positions.
+
+.. image::  images/results_2.png
+    :alt: The geometry
+    :align: center
+    :name: results
+    :width: 600
+
+
+
+
+
+
+References
+-----------
+
+[1] Mirakhori, G., Collard-Daigneault, A., Alphonius, A., Doucet, J., Blais, B., & Chaouki, J. (2023a). An improved position reconstruction method for radioactive particle tracking. *Nuclear Instruments and Methods in Physics Research*. 168504. https://doi.org/10.1016/j.nima.2023.168504
```

Comment:
```suggestion
`[1] <https://doi.org/10.1016/j.nima.2023.168504>`_  Mirakhori, G., Collard-Daigneault, A., Alphonius, A., Doucet, J., Blais, B., & Chaouki, J. (2023a). An improved position reconstruction method for radioactive particle tracking. *Nuclear Instruments and Methods in Physics Research*. 168504. 
```
This will put it in hyperlink